spec/factories/user_counters.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user_counter do
+    user_id { 1 }
+    counter_name { "MyString" }
+    count { 1 }
+    decayed_count { 1.5 }
```

Comment:
in reality you should never have a `decayed_count` greater than the corresponding `count`. But not that it really matters for any tests.